docs/examples/notebooks/06-chaotic-attractor.ipynb

```diff
@@ -224,18 +228,69 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "fig, ax, anim = make_anim_chaotic(sim.data[output_p][:, 0],\n",
-    "                                  sim.data[output_p][:, 1],\n",
-    "                                  sim.data[output_p][:, 2],\n",
-    "                                  ms_per_frame=20)\n",
-    "HTML(anim.to_html5_video())"
+    "def make_anim():\n",
+    "    _, _, anim = make_anim_chaotic(sim.data[output_p][:, 0],\n",
+    "                                   sim.data[output_p][:, 1],\n",
+    "                                   sim.data[output_p][:, 2],\n",
+    "                                   ms_per_frame=20)\n",
+    "    return anim\n",
+    "\n",
+    "\n",
+    "HTML(make_anim().to_html5_video())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 9: Spiking Neurons\n",
+    "\n",
+    "The plots above show the output of the Lorenz attractor network implemented with non-spiking rectified linear neurons. The network can also be simulated using spiking neurons to illustrate the similarities and differences between a spiking and a non-spiking network.\n",
+    "\n",
+    "Below, we configure the FPGA neural ensemble to use spiking neurons, run the simulation, and plot the results."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with model:\n",
+    "    fpga_ens.ensemble.neuron_type = nengo.SpikingRectifiedLinear()\n",
+    "\n",
+    "with nengo_fpga.Simulator(model) as sim:\n",
+    "    sim.run(20)\n",
+    "plot_results()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "HTML(make_anim().to_html5_video())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The plots above show that with spiking neurons, the output of the network is, expectedly, more noisy (less precise) than the results of the rate-mode network. However, despite this, the chaotic attractor network in its current configuration is stable, and exhibits the expected behaviour of the Lorenz attractor.\n",
+    "\n",
+    "It should be noted that the output of the spiking Lorenz attractor network differs from that of the non-spiking network because of two factors. First, every time the Nengo simulator is created, the network parameters are regenerated (randomized). Second, because the dynamics of the Lorenz attractor are extremely sensitive to the initial conditions of the network (which is what makes it chaotic), even if the spiking and non-spiking networks were generated with the same parameters, the additional noise introduced by the spikes cause the spiking output to diverge from the trajectory of the non-spiking network."
```

Comment:
Still a "rate-mode" here

Reply:
That's really weird. I did a search in both the text editor and in the built HTML docs file...